internal/wireguard/settings.go

```diff
@@ -45,6 +45,9 @@ type Settings struct {
 	// Implementation is the implementation to use.
 	// It can be auto, kernelspace or userspace, and defaults to auto.
 	Implementation string
+	// PersistenKeepAlive enable sending keep alive signal to server.
+	// So NAT view it as active connection
```

Comment:
// PersistenKeepAlive enables sending keep alive signals to the Wireguard 
	// server so NAT views it as an active connection. It defaults to 0, which means
    // it is not to be used.